File: problems/242. Valid Anagram/Solution.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        sLetters, tLetters = [0] * 26, [0] * 26
        a = ord('a')
        for char in s:
            sLetters[ord(char) - a] += 1
        for char in t:
            tLetters[ord(char) - a] += 1
        for i in range(0, 26):
            if sLetters[i] != tLetters[i]: return False
        return True

File: problems/242. Valid Anagram/Solution.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length()) return false;
        int sLetters[26], tLetters[26];
        for (char& ch : s) sLetters[ch - 'a'] += 1;
        for (char& ch : t) tLetters[ch - 'a'] += 1;
        for (int i = 0; i < 26; i++) if (sLetters[i] != tLetters[i]) return false;
        return true;
    }
};